packages/widgetbook/test/src/knobs/knobs_test.dart

```diff
@@ -0,0 +1,75 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:provider/provider.dart';
+import 'package:widgetbook/src/knobs/knobs.dart';
+import 'package:widgetbook/src/repositories/selected_story_repository.dart';
+import 'package:widgetbook/widgetbook.dart';
+
+import '../../helper/widget_test_helper.dart';
+
+void main() {
+  testWidgets(
+    'Bool knob added',
+    (WidgetTester tester) async {
+      final selectedStoryRepository = SelectedStoryRepository();
+      final knobsNotifier = KnobsNotifier(selectedStoryRepository);
+      final useCase = WidgetbookUseCase(
+        name: 'use case',
+        builder: (context) {
+          return Column(
+            children: [
+              Text(context.knobs.boolean(
+                label: 'label',
+                initial: true,
+              )
+                  ? 'Hi'
+                  : 'Bye'),
+              ...knobsNotifier.all().map((e) => e.build())
+            ],
+          );
+        },
+      );
+      selectedStoryRepository.setItem(useCase);
+      await tester.pumpWidgetWithMaterialApp(ChangeNotifierProvider(
+        create: (context) => knobsNotifier,
+        child: Builder(builder: useCase.builder),
```

Comment:
i think it would be better to pass the selected usecase as a constructor parameters when the repository is created instead of calling `selectedStoryRepository.setItem(useCase)`. Because calling `setItem` will partially test the `setItem` method as well.

Reply:
There doesnt seem to be a constructor parameter to selected story repository, is this something you'd like me to add?

Reply:
Yes. Please add this functionality.